index.bs

```diff
@@ -1136,26 +1134,20 @@ a [=/URL=] |url|,
 [=/scalar value string=] |path|, and
 [=/boolean=] |partitioned|:
 
-1. Let |expires| be the earliest representable date represented [=as a timestamp=].
-
-    Note: The exact value of |expires| is not important for the purposes of this algorithm,
-    as long as it is in the past.
-
+1. Let |maxAge| be 0.
```

Comment:
I think we can just inline this as we do for the others. (It's slightly less clear, but it's more consistent.)